src/Stripe.net/Entities/Issuing/Authorizations/Authorization.cs

```diff
@@ -32,16 +38,11 @@ public class Authorization : StripeEntity<Authorization>, IHasId, IHasMetadata,
         [JsonProperty("authorization_method")]
         public string AuthorizationMethod { get; set; }
 
-        /// <summary>
-        /// The amount that has been authorized. This will be 0 when the object is created, and
-        /// increase after it has been approved.
-        /// </summary>
+        [Obsolete("This field is considered deprecated.")]
```

Comment:
If we're not pointing to a replacement field/method, I think we can simply tag the field as `[Obsolete]` without a message. The current message does not provide any useful information.

Reply:
Ack fair. It's going away tomorrow so I'll keep as is but will do that next time